### DISCUSSION TITLE
What is the license on the code and data in this repository?

### DISCUSSION THREAD

**Discussion by bradlarsen** [digipres/digipres.github.io#42] [Q&A] [answered]
Hi there. Great website!

I am working on adding some content classification capabilities to [Nosey Parker](https://github.com/praetorian-inc/noseyparker), a tool that detects secrets and sensitive information in textual data.

I haven't found classification tools that exactly fit my needs, so I'm looking at building some of my own.

I see that you've already put together a comprehensive aggregated list of extension -> file type from a number of other tools: https://github.com/digipres/digipres.github.io/blob/master/_data/formats/extensions.yml.

What is the license on that?

Thanks, 
Brad Larsen

**Comment by carlwilson** [comment, marked as answer]
Hi @bradlarsen, good question. I'd guess it should be a Creative Commons with an attribution license, but there's nothing explicitly stated.

**Comment by anjackson** [comment]
Good point, we should clearer about the licensing. If these extracts count as derivative works, this will mean going back and checking the terms of each source. 

The code that generates the aggregates is Apache licensed, if that helps: https://github.com/digipres/sentinel

The thing is, this project wasn't designed with onward re-use of the data in mind.  It's purpose was to aggregate the data so people could find their way back to the source registry, while exposing and conflicts and inconsistencies for review and resolution.  As such, it's not clear how useful or trustworthy the aggregated data is for other purposes.

**Reply by bradlarsen** [reply]
Thank you for clarifying. I will investigate the upstream sources further.
